scripts/build/builders/host.py

```diff
@@ -45,21 +46,61 @@ def BinaryName(self):
             raise Exception('Unknown app type: %r' % self)
 
 
-def ConcretePlatformName():
-    uname_result = uname()
-    return '-'.join([uname_result.system.lower(), release(), uname_result.machine])
+class HostBoard(Enum):
+    NATIVE = auto()
+
+    # cross-compile support
+    CROSS_COMPILE_ARM64 = auto()
+
+    def PlatformName(self):
+        if self == HostBoard.NATIVE:
+            uname_result = uname()
+            return '-'.join([uname_result.system.lower(), uname_result.machine])
```

Comment:
Please check machine, if it's x86_64, label it something like linux-x64. Otherwise bail.

The name should be the same as the name used when cross compiling an x64 binary from the Pi. Compiling is symmetrical; it's is _how_ you build not _what_ you are building. This label should be the 'what' and it should ignore the 'how'.

Reply:
Updated.  In theory this may also compile 'darwin'so I left code logic a bit different. I allow cross compile if the 'native' platform is not the same as the cross compile.